pkg/manager/manager_test.go

```diff
@@ -133,8 +135,8 @@ func Test_LoadManager(t *testing.T) {
 		"../../configs/configStore-sample.json",
 		"../../configs/changeStore-sample.json",
 		"../../configs/networkStore-sample.json",
-		NewMockDeviceChangesStore(gomock.NewController(t)),
-		NewMockNetworkChangesStore(gomock.NewController(t)))
+		mockstore.NewMockDeviceChangesStore(gomock.NewController(t)),
+		mockstore.NewMockNetworkChangesStore(gomock.NewController(t)))
```

Comment:
coudl we reuse the same controller as we do at lines 100/101/102 ? I believe this creates less objects